content/general/concepts/open-source/open-source.md

```diff
@@ -0,0 +1,26 @@
+---
+Title: 'Open Source'
+Description: 'Open source refers to software that is released with the source code made freely available for modification and possible redistribution.'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags:
+  - 'Conceptual'
+  - 'Development'
+  - 'Linux'
+CatalogContent:
+  - 'paths/code-foundations'
+  - 'paths/computer-science'
+---
+
+The term "open source" refers to softwares that are freely available for modification and possible redistribution. Open source projects allow the source code to be reused by anyone with the restriction that any derivative projects are released under the same license.
```

Comment:
I had to look this up but apparently "softwares" is not a word and the plural of software is software. I have been enlightened! 🙏

```suggestion
The term "open source" refers to software that is freely available for modification and possible redistribution. Open source projects allow the source code to be reused by anyone with the restriction that any derivative projects are released under the same license.
```